knowledge-base/how-to-set-the-map-extent-dynamically-when-showing-a-single-data-point.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Set Map extent dynamically when showing a single data point
+description: Explains how to set map extent dynamically when showing a single data point
+type: how-to
+page_title: How to Set the Map Extent Dynamically When Showing a Single Data Point
+slug: how-to-set-the-map-extent-dynamically-when-showing-a-single-data-point
+position: 
+tags: map
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
```

Comment:
There is no point in adding only a single entry of Product in the environment table. Add all other relevant fields here, for example that we are talking about a report component/widget - the Map, etc..